tests/x11/validate_vnc_from_client.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that after remote installation, the VNC connection
```

Comment:
> it would be better to retrieve the ip from the `/var/lib/dhcp/db/dhcpd.leases` and handle it via code than having a hardcoded ip via testdata. Let me know if you need any help

I agree because that wouldn't allow network change to break the test. But from what I have checked so far, there are a lot of openqa tests that use the leased IP hardcoded. Not sure which is the best way to go here. I already have the code for this and changed it to retrieve the last leased address as Joaquin pointed out in one of the comments